apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -26,99 +26,85 @@ export function Lti13CourseNavigationInstructor({
-                  <div class="form-group">
-                    <label class="form-label" for="connect_course_instance">
-                      Connect ${courseName} with:
+                <div class="mb-3">
+                  <form method="POST" id="link_form">
+                    <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+
+                    <label class="form-label" for="connect_course">
+                      Select a PrairieLearn course:
                     </label>
                     <select
-                      class="custom-select"
-                      id="connect_course_instance"
-                      name="unsafe_course_instance_id"
+                      id="connect_course"
+                      class="form-select mb-3"
+                      name="unsafe_course_id"
+                      hx-get="/pl/lti13_instance/${lti13_instance_id}/course_navigation/course_instances"
+                      hx-include="#link_form"
+                      hx-target="#course_instances"
+                      required
                     >
-                      <option value="" disabled selected>
-                        Select an existing course instance...
-                      </option>
-                      ${courses.map((course) => {
-                        const course_cis = course_instances.filter(
-                          (ci) => ci.course_id === course.id,
-                        );
-                        return html`
-                          <optgroup label="${course.short_name}: ${course.title}">
-                            ${course_cis.map((ci) => {
-                              return html`<option value="${ci.id}">
-                                ${course.short_name} -- ${ci.long_name} (${ci.short_name})
-                              </option>`;
-                            })}
-                          </optgroup>
-                        `;
+                      <option selected disabled value="">Select a course to continue</option>
+                      ${courses.map((c) => {
+                        return html`<option value="${c.id}">${c.short_name}: ${c.title}</option>`;
                       })}
                     </select>
-                  </div>
-                  <button class="btn btn-primary" id="saveButton" disabled>Save</button>
-                </form>
+                    <label class="form-label" for="course_instances">
+                      Select a course instance to connect:
+                    </label>
+                    <select
+                      id="course_instances"
+                      name="unsafe_course_instance_id"
+                      class="form-select mb-3"
+                      required
+                    >
+                      <option selected disabled value="">See above</option>
+                    </select>
+                    <input type="submit" class="btn btn-primary" value="Connect courses" />
```

Comment:
Ha, I always do a double-take when I see this instead of a `<button>`. Is there any reason you use this instead? I know it works, but it feels a little idiosyncratic in our codebase.

Nit: "Connect course instance" as the label instead?

Reply:
I think it's just a habit of mine when I'm writing inputs. Updated to a button in 633032758